README.md

```diff
@@ -244,7 +248,13 @@ Hi COURTOISON. Today is Tuesday, tomorrow is Wednesday.
 - **image(width, height)**: Transform an url string to an image
 - _More to come..._
 
-_Warning: If you use an undefined formatter it will be simply ignored, which could lead to unexpected results if you're chaining them._
+_Warning: If you use an undefined formatter it will be simply ignored, which could lead to unexpected results if you're chaining them. See [Strict mode](#strict-mode)_
+
+### Strict mode
+
+By default, _google-docs-mustaches_ is failing safely, which means that you don't have to worry about using an undefined variable or a unknow formatter, the generated errors will be catched and treated by the program itself. However, you might face unexepected behaviour if for example, you chain several formatters and one of them is misspelled, it would be ignored and the output of your formatters pipeline won't match your expectations.
+
+To avoid this, we provide a strict mode for `.interpolate` and `.discovery`. Instead of using an empty string or your fallback when encountering an undefined variable, it will throw an exception, aborting immediatly the interpolation of your document. 
```

Comment:
Should we rename `discovery` to `discover`?

Reply:
I'd do it in a "pre-release refactoring", it's not really the point of this PR, right?

Reply:
I'm fine with that